docs/editor_guide/creating_segments.rst

```diff
@@ -0,0 +1,65 @@
+Creating a segment
+==================
+
+To create a segment, go to the "Segments dashboard" and click "Add segment".
+You can find the segments dashboard in the administration menu on the left of
+the page.
+
+.. figure:: ../_static/images/segment_dashboard_header.png
+   :alt: The segment dashboard header containing the "Add segment" button.
+
+On this page you will be presented with two forms. One with specific information
+about your segment, and the other allowing you to choose and configure your
+rules.
+
+
+Set segment specific options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: ../_static/images/edit_segment_specifics.png
+   :alt: The form that allows you to configure specific segment options.
+
+1. Enter a name for your segment.
+
+    Choose something meaningful like "Newsletter campaign visitors". This will
+    ensure you'll have a global idea about which visitors are in this segment in
+    other parts of the administration interface.
+
+2. Select the status of the segment *Optional*
+
+    You will generally keep this one on **enabled**. If for some reason you wan't
```

Comment:
Geen idee of het uitmaakt maar je gebruikt wel ``You'll`` en ``You will`` door elkaar..